metrics/ci_integrations/lib/integration/ci/config/model/sync_config.dart

```diff
@@ -12,18 +13,32 @@ class SyncConfig extends Equatable {
   /// the loaded metrics will be saved to.
   final String destinationProjectId;
 
+  /// A number of builds to fetch from the source during project's
```

Comment:
Would it be great to update the class documentation since the implementation changed?